### ISSUE TITLE
Intel Undervolting

### ISSUE THREAD

**Issue by simoneruffini** [delta-one/dell-xps-15-9570#5]
With the new bios updates, are you still able to undervolt your CPU?

**Comment by delta-one** [comment]
`intel-undervold read` gives me the following output:
```
CPU (0): -150.39 mV
GPU (1): -0.00 mV
CPU Cache (2): -150.39 mV
System Agent (3): -0.00 mV
Analog I/O (4): -0.00 mV
```
But if I remember correctly, I had to reset the BIOS after upgrading to 1.16 or whatever it was as explained [here](https://github.com/kitsunyan/intel-undervolt/issues/43#issuecomment-618412498).

**Comment by simoneruffini** [comment]
So you're saying that you can undervolt your system even with the new bios updates? Or are you holding them off to keep this feature?

**Comment by delta-one** [comment]
I'm running the newest 1.19.0.

**Comment by simoneruffini** [comment]
Hmm, I read that post but from what I understood new firmware updates disabled that feature, so if you want it back you need to downgrade. But you are telling me that you can undervolt with the new firmware and to do that you just restored to factory settings the BIOS and nothing else?

**Comment by delta-one** [comment]
Exactly. You'll have to reset the BIOS to factory settings after the upgrade and then you can undervolt again (this is needed only once).

**Comment by simoneruffini** [comment]
I tried today but it did not fix the problem. To be more precise I reset the bios to factory settings by pressing ALT+F and then rebooting the machine. After the reboot I quickly re-entered the boot menu and configured some things that were not like I want them (fn keys and touchscreen primarly). Then once that was done, I booted in linux and tried undervolt. Didn't work, same `values do not equal`. 
In your case how did you reset your system? Did you restored some settings or kept everything as is after the reset?

**Comment by delta-one** [comment]
I don't remember the details, but I reset it somewhere in the BIOS, probably like described in the other thread. I also changed a thing or two afterwards again.
